liquibase-extension-examples/pom.xml

```diff
@@ -55,7 +55,7 @@
           <plugin>
               <groupId>org.apache.felix</groupId>
               <artifactId>maven-bundle-plugin</artifactId>
-              <version>3.3.0</version>
```

Comment:
@nvoxland, hi! This is a significant jump in version for the liquibase-extension-examples. Can you check and see if we have any tests that show this does not break anything?

Reply:
This is just an example extension that's not really used for anything at this point. I'll open a ticket to look at deleting it, but for now keeping the example up to date is fine.

Reply:
Thank you for the clarification, @nvoxland .